src/epoch.rs

```diff
@@ -2827,12 +2825,11 @@ impl Epoch {
 impl Epoch {
     /// Initializes a new Epoch from `now`.
     /// WARNING: This assumes that the system time returns the time in UTC (which is the case on Linux)
-    /// Uses [`std::time::SystemTime::now`](https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.now) under the hood
+    /// Uses [`std::time::SystemTime::now`](https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.now)
+    /// or javascript interop under the hood
     pub fn now() -> Result<Self, Errors> {
-        match SystemTime::now().duration_since(SystemTime::UNIX_EPOCH) {
-            Ok(std_duration) => Ok(Self::from_unix_seconds(std_duration.as_secs_f64())),
-            Err(_) => Err(Errors::SystemTimeError),
-        }
+        let duration = crate::system_time::duration_since_unix_epoch()?;
```

Comment:
Ah, this needs to be "from unix duration" now, otherwise it assumes that the reference epoch is J1900 (the TAI ref epoch), but unix seconds are wrt 01 January 1970

Reply:
I could not find a `from_unix_duration()`. Should I add it?